server/cluster.go

```diff
@@ -120,6 +137,18 @@ func (c *RaftCluster) loadBootstrapTime() (time.Time, error) {
 	return typeutil.ParseTimestamp([]byte(data))
 }
 
+func (c *RaftCluster) initCluster() {
```

Comment:
Rename `RaftCluster` to `Cluster`?

Reply:
As for the naming problem, I think we can leave it to another PR.